### PR TITLE
Update IdentifierUris of ADApp and DisplayName of ADServicePrincipal

### PR DESCRIPTION
**Pull request by bganapa**
cherry-picking and fixing commit 2281c88c80c069c7f1776392788f83366cdfb4b3, PR-14952

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Resources/Resources/ChangeLog.md [hunk 1]
```diff
@@ -19,7 +19,11 @@
 -->
 ## Upcoming Release
 
-## Version 1.10.0
+## Version 0.12.0
```
Comment (dingmeng-xue) [inline src/Resources/Resources/ChangeLog.md:22]: I forgot whether we need to bump the module version during publishing module from this branch. If bump version is required, you should not add version here.

Comment (bganapa) [reply src/Resources/Resources/ChangeLog.md:22]: I have updated the versions now with this commit , https://github.com/Azure/azure-powershell/pull/16247/commits/9a63d94a2ebddcebae062c398c38b6de66deb035

In this branch we dont bump version automatically, it is manual

**Review by dingmeng-xue — APPROVED** [review]

**Pull request merged**